<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Run_PopupClass</class>
 <widget class="QMainWindow" name="Run_PopupClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run as...</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="trainingCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Run in Training Mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pcapCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Read packets from file</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="pcapGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Packet File Options</string>
    </property>
    <widget class="QTextEdit" name="pcapEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>248</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="pcapLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Packet Capture File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pcapButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>57</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="liveCapCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Go to live capture after reading</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
